packages/iov-tendermint-rpc/src/encodings.ts

```diff
@@ -44,8 +44,21 @@ export function may<T, U>(transform: (val: T) => U, value: T | null | undefined)
   return value === undefined || value === null ? undefined : transform(value);
 }
 
-export function parseInteger(str: IntegerString): number {
-  return Int53.fromString(str).toNumber();
+export class Integer {
+  public static parse(str: IntegerString): number {
+    return Int53.fromString(str).toNumber();
+  }
+
+  public static encode(num: number): IntegerString {
+    return new Int53(num).toString() as IntegerString;
+  }
+
+  public static ensure(n: number): number {
```

Comment:
If you don't trust the runtime type to be the type system type, this shout change to `n: unknown`.

Also since this is called Integer.ensure, we need at least a [Number.isInteger](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger) test

Reply:
Yeah, the docs say it is a string, but in tests it was an integer.
I wanted to make sure it was there